src/v/pandaproxy/client/test/consumer_group.cc

```diff
@@ -57,6 +58,24 @@
 
 namespace ppc = pandaproxy::client;
 
+namespace {
+auto offset_request_from_assignment(ppc::assignment assignment) {
```

Comment:
why not define the return type. Seems like the existing style prefers explicit type sigs at the function boundary level.

Reply:
Done